### PR TITLE
Added cylinder 3D primitive to Workplane.

### PR DESCRIPTION
**Pull request by martinbudden**
I've added a cylinder 3D primitive to the `Workplane`, as per item on the roadmap, https://github.com/CadQuery/cadquery/blob/master/doc/roadmap.rst

My first submission to cadquery so a review is very much appreciated, and I'm happy to make any changes requested.

### CODE REVIEW

**Review by marcus7070 — APPROVED** [review]
Looks great, thanks @martinbudden!

File: cadquery/cq.py [hunk 1]
```diff
@@ -3800,6 +3800,69 @@ def sphere(
...
+            will extend in the positive x, y and z directions. Can also use a 3-tuple to specify
+            centering along each axis.
+        :param combine: Whether the results should be combined with other solids on the stack
+            (and each other)
+        :type combine: true to combine shapes, false otherwise
+        :param clean: call :py:meth:`clean` afterwards to have a clean shape
+        :return: A cylinder object for each point on the stack
+
+        One cylinder is created for each item on the current stack. If no items are on the stack, one
+        box using the current workplane center is created.
```
[the remark this answers is not in the document]
Comment (martinbudden) [reply cadquery/cq.py]: Fixed.

File: tests/test_cadquery.py [hunk 2]
```diff
@@ -2443,6 +2443,40 @@ def testSphereCombine(self):
         self.assertEqual(1, s.solids().size())
         self.assertEqual(4, s.faces().size())
 
+    def testCylinderDefaults(self):
+        s = Workplane("XY").cylinder(20, 10)
+        self.saveModel(s)
```
[the remark this answers is not in the document]
Comment (martinbudden) [reply tests/test_cadquery.py]: Fixed.

**Review by jmwright — APPROVED** [review]
Thanks @martinbudden

**Pull request merged**